6001-7000/LIT6341CCR34.xml

```diff
@@ -78,7 +78,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                 <lb/>መስቀል፡ ወለዩ፡ ለያፍቅረነ፡ እግዚእ። ወያፍቅረነ፡ እግዚእ፡
                 <lb/>ወለዩ፡ ለመሐሬ፡ እግዚእ፡ ወመሐሪ፡ እግዚእ፡ ወለደ፡
             </ab></div>
-            
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="saws:isDifferentTo" active="LIT6341CCR34" passive="LIT3143Ledata"><desc>For the first three 
```

Comment:
I think the relation should define CCR34 is a version of LIT3143Ledata

Reply:
OK. Is it better to use saws:isCloseRenderingOf or any other relation?

Reply:
We have saws:isVersionOf